Makefile

```
@@ -80,6 +80,9 @@ lint-front: ## Lint the Javascript code
 clean-front: ## Clean the frontend builds
 	yarn run clean
 
+list-outdated-front: ## List outdated Node.js packages
+	@npx david
```

Quand il y a des dépendances qui ne sont pas à jour, `npx david` renvoie le code 1, ce que `make` n'aime pas : j'ai à la fin de la sortie de david : `make: *** [list-outdated-front] Error 1`.

Je propose : 
```suggestion
	@npx david || true
```

Sinon OK.

---

Bonne idée ! J'ai mis à jour la PR